pkg/operation/botanist/component/istio/istiod_test.go

```diff
@@ -1937,7 +1937,7 @@ spec:
           app: aggregate-prometheus
           role: monitoring
     ports:
-    - port: metrics
+    - port: 15014
```

Comment:
Cosmetic suggestion to keep it idiomatic with the other prometheus netpols:
Lets preserve the named port here and add the following port to the istiod Pod template:
```yaml
          - containerPort: 15014
            name: metrics
            protocol: TCP
```

Reply:
I'm a bit hesitant here, as we have the deployment as chart that we have to update from time to time with upstream istio charts. 
Should we really add this additional diff?
@ScheererJ, what do you think?

Reply:
Ah, okay. if the upstream manifest does not have the port, then let's keep it as it is. 👍